src/app/(routes)/across-bridge/_components/testimonials-section.tsx

```diff
@@ -0,0 +1,273 @@
+"use client";
+
+import Image from "next/image";
+import { useState } from "react";
+
+import { TwitterIcon, VerifiedCheckmarkIcon } from "@/app/_components/icons";
+import { Text } from "@/app/_components/text";
+import { twMerge } from "@/app/_lib/tw-merge";
+
+const mainTweet = {
+  title: (
+    <>
+      Lowest fees, Zero slippage,
+      <br />
+      professional team.
+    </>
+  ),
+  body: (
+    <>
+      &apos;I&apos;ve tried every L2 bridge out there. @AcrossProtocol is undoubtedly the
+      best one.
+      <br />
+      <br />
+      Lowest fees, 0 slippage, professional team. Very under-appreciated, use it.&apos;
+    </>
+  ),
+  username: "@aradtski",
+};
+
+const tweets = {
+  firstColumn: [
+    {
+      isVerified: true,
+      displayName: "pray.eth",
+      username: "@pray_eth",
+      profileImageUrl:
+        "https://pbs.twimg.com/profile_images/1692052698307153920/WiHp_6vw_200x200.jpg",
+      tweet: (
+        <>
+          Yeah, I bridge a lot, noticed @BungeeExchange routes were giving me
+          @AcrossProtocol most of the time, so just started to use it as my default bridge
+          (minimizing risk is my primary concern when bridging).
+          <br />
+          <br />
+          Transfers are incredibly fast and cheap. Great work!
+        </>
+      ),
```

Comment:
Nit: I'm not a huge fan of having markdown for each individual tweet. Not sure if there's a better solution, but it feels like it may be unwieldy if we increase the number of tweets.

Reply:
Yea, I am also not a big fan of this. But that's the best I could come up with. Especially with the Twitter API being highly limited (free version at least)